### ISSUE TITLE
Missing babel-runtime?

### ISSUE THREAD

**Issue by schmoove** [glazedio/frisbee#15]
Is there something I'm missing after upgrading to 1.0.5 with the new "import" syntax? Continually getting this error now:

```
Unable to resolve module babel-runtime/regenerator from /Users/Mac/Desktop/Mobile/node_modules/frisbee/lib/frisbee.js: Unable to find this module in its module map or any of the node_modules directories under /Users/node_modules/babel-runtime/regenerator and its parent directories
```

Help!

**Comment by niftylettuce** [comment]
Taking a look

**Comment by niftylettuce** [comment]
you're correct @schmoove, I'm releasing `1.1.0` and deprecating `< 1.1.0`.  Thanks for this critical bug find.

**Comment by niftylettuce** [comment]
published to NPM version 1.1.0, try it out and let me know if this resolves your issue @schmoove
